wecs/mechanics/clock.py

```diff
@@ -30,6 +31,18 @@ class Clock:
     game_time: float = 0.0
 
 
+@Component()
+class Calendar:
+    year:   int = 3000
+    #                             list(current, max)
+    month:  list = field(default_factory=lambda: list((1,12)))
+    day:    list = field(default_factory=lambda: list((1,30)))
+    hour:   list = field(default_factory=lambda: list((12,24)))
+    minute: list = field(default_factory=lambda: list((0,60)))
+    second: list = field(default_factory=lambda: list((0,60)))
```

Comment:
A bit of documentation may be in order, in particular on the numbers used. Without a deeper look, the 12 especially smells weird.

Reply:
Also, this is a rather eurocentric calendar. That sure can be a bit more abstract?